[website] fix Footer elements appearance while in error mode

Simek opened the pull request:
# Why

Currently, when error mode is activated red background is set to the `EditorFooter` and fonts colors are altered. This is causing some display issues when user hovers or activates a Footer button or SDK Switcher.

<img width="534" alt="Screenshot 2021-02-23 at 14 16 43" src="https://user-images.githubusercontent.com/719641/108848853-c8affc00-75e1-11eb-99e2-d61bf8f6c4bb.png">

# How

This PR adds few styles overwrites for states like `hover` and `active` to retain readability even in error mode. This probably can be improved in future utilizing the `hover` color with an alpha value (transparency).

Additionally in this PR I have fixed the `select` arrow character position and I have updated the `select` background so it's consistent with latest tweak for the `Togglers`.

Last small change fixes the `Toast` button colors in Dark Mode.

# Test Plan

I have tested the changes on localhost and have utilized Windows and macOS device to confirm that style changes are correct and don't cause an issues on one of the platforms.

## Preview

### Windows

<img width="408" alt="Screenshot 2021-02-23 141045" src="https://user-images.githubusercontent.com/719641/108848300-1d9f4280-75e1-11eb-8cb9-b433d9951b3d.png">

<img width="488" alt="Screenshot 2021-02-23 141250" src="https://user-images.githubusercontent.com/719641/108848396-3d366b00-75e1-11eb-931b-7ea7bcda7fc8.png">

<img width="308" alt="Screenshot 2021-02-23 141329" src="https://user-images.githubusercontent.com/719641/108848479-52ab9500-75e1-11eb-8930-ef79a9350ece.png">

### macOS

<img width="534" alt="Screenshot 2021-02-23 at 14 00 55" src="https://user-images.githubusercontent.com/719641/108848615-7c64bc00-75e1-11eb-82a1-b46b0f21b474.png">

<img width="534" alt="Screenshot 2021-02-23 at 14 00 49" src="https://user-images.githubusercontent.com/719641/108848680-930b1300-75e1-11eb-81af-a279330afa6a.png">

IjzerenHein reviewed (changes requested):
Thanks 😃 

The toast en footer hover changes look great!

The SDK version switcher doesn't look right to me tbh. When the bottom-bar is blue or red, it stands out way too much. I think we should keep its background color the same as the bar, and only change the text-color when the bottom bar is blue or red.

Simek commented:
@IjzerenHein I have reverted SDK Switcher background color to transparent as you suggested and I have added a code to ensure the select border have the correct color in the error mode (previously, when Dark Mode was on, select have dark border, which do not match the overall error styling).

#### Preview
<img width="397" alt="Screenshot 2021-02-25 230134" src="https://user-images.githubusercontent.com/719641/109225585-c1dde080-77bd-11eb-846d-cb5e7a0ec113.png">
<img width="384" alt="Screenshot 2021-02-25 230112" src="https://user-images.githubusercontent.com/719641/109225592-c3a7a400-77bd-11eb-9b39-023d7f22bd18.png">

IjzerenHein reviewed (approved):
Looks great, thanks 😄
